stock_eggs/source-engine/egg-counter--strike--global-offensive.json

```diff
@@ -49,6 +54,96 @@
             "user_viewable": false,
             "user_editable": false,
             "rules": "required|string|max:20"
+        },
+        {
+            "name": "Ticrate",
+            "description": "Set server ticrate to 64 or 128",
+            "env_variable": "TICRATE",
+            "default_value": "128",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|integer|in:64,128"
+        },
+        {
+            "name": "Player Slots",
+            "description": "No. of player slots",
+            "env_variable": "MAXPLAYERS",
+            "default_value": "10",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|integer|digits_between:1,20"
+        },
+        {
+            "name": "Game Type",
+            "description": "Game Type: 0 = Classic, 1 = Gun Game, 2 = Training, 3 = Custom, 4 = Cooperative, 5 = Skirmish, 6 = Free For All",
+            "env_variable": "GAMETYPE",
+            "default_value": "0",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|digits_between:0,6"
+        },
+        {
+            "name": "Game Mode",
+            "description": "Game Mode: 0= Casual, 1 = Competitive, 2 = Wingman",
+            "env_variable": "GAMEMODE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|digits_between:0,2"
+        },
+        {
+            "name": "Map Group",
+            "description": "Set server map group: mg_active for competetive",
```

Comment:
Are there other groups that could be added to the list here?

Reply:
yes, If its there ill add in next.